neural_speed/core/CMakeLists.txt

```diff
@@ -36,9 +36,10 @@ endif()
 if(NOT WIN32)
   target_link_libraries(ne_layers PUBLIC rt)
 else()
-  target_link_options(ne_layers PUBLIC /STACK:5242880)
+  target_link_options(ne_layers PUBLIC /STACK:5242880 /F5242880)
```

Comment:
Sorry, what does `/STACK:5242880 /F5242880` mean?

Reply:
enlarge the stack size, /STACK for MSVC, /F for DPCPP